### DISCUSSION TITLE
Hide login form on login

### DISCUSSION THREAD

**Discussion by phh** [filamentphp/filament#11511] [General]
I'm adding Socialite buttons to my login form, as I want my users to use X Socialite provider to login and not the default user/password. The buttons are added via `PanelsRenderHook::AUTH_LOGIN_FORM_AFTER` which works fine. However, I cannot disable the login form since this is a "hardcoded" blade component:

```blade
<x-filament-panels::page.simple>
    ...

    {{ \Filament\Support\Facades\FilamentView::renderHook(\Filament\View\PanelsRenderHook::AUTH_LOGIN_FORM_BEFORE, scopes: $this->getRenderHookScopes()) }}

    <x-filament-panels::form wire:submit="authenticate">
        {{ $this->form }}

        <x-filament-panels::form.actions
            :actions="$this->getCachedFormActions()"
            :full-width="$this->hasFullWidthFormActions()"
        />
    </x-filament-panels::form>

    {{ \Filament\Support\Facades\FilamentView::renderHook(\Filament\View\PanelsRenderHook::AUTH_LOGIN_FORM_AFTER, scopes: $this->getRenderHookScopes()) }}
</x-filament-panels::page.simple>
```

I could always publish the view and override it, however, I'm really fan of the renderhooks idea to avoid overwriting any templates.

Any ideas?

**Comment by leandrocfe** [comment]
You could create a new simple page to use the Socialite and hide the default login.
https://github.com/filamentphp/filament/discussions/11370#discussioncomment-8471323
